src/ManageDatesView.elm

```diff
@@ -33,44 +33,54 @@ view model =
             model.pickedDates
                 |> List.indexedMap
                     (\i d ->
-                        div []
-                            [ input
-                                [ class "form-control pikaday-input"
-                                , id ("pikaday-instance-" ++ (toString i))
-                                , value (displayDateWithoutTime d)
-                                , disableInput d
+                        div [ class "form__question-section form__question-section--table" ]
+                            [ div [ class "form__question-sub-section form__question-sub-section--table" ]
+                                [ label [ class "form__label" ]
+                                    [ text "Date" ]
+                                , input
+                                    [ class "form__input  pikaday-input"
+                                    , id ("pikaday-instance-" ++ (toString i))
+                                    , value (displayDateWithoutTime d)
+                                    , disableInput d
+                                    ]
+                                    []
                                 ]
-                                []
-                            , button
-                                [ onClick (DeleteDate d)
-                                , style [ ( "margin-left", "0.2rem" ) ]
-                                , disableInput d
+                            , div [ class "form__question-sub-section form__question-sub-section--table form__question-sub-section__button" ]
+                                [ button
+                                    [ onClick (DeleteDate d)
+                                    , disableInput d
+                                    , class "button button--secondary icon icon--bin"
+                                    ]
+                                    []
                                 ]
-                                [ text "Delete" ]
                             ]
                     )
 
         column1 =
-            div [ class "form-group" ]
-                [ div [ class "input-group" ]
+            div []
+                [ span [ class "form__hint" ]
+                    [ text "You will be unable to change any dates that have sessions" ]
```

Comment:
Please can we include this icon http://fontawesome.io/icon/exclamation-circle/
1. into the icon namespace that already exists:
    ( `.icon` `.icon--warning`)
    between the lines that feature:
```css
.icon--success:before { content: '\e801'; } /* '' */
.icon--failure:before { content: '\e802'; } /* '' */

And then at the beginning of this span (probably nested inside the span) as an `<i class="[etc]">`


------

Reason being, At the moment I think this looks much like a description or instruction... until you read it. We're going for an "immediately obvious what's what" approach so I think a bit of colour and an icon will go a long way to telling the user that they should mind this message and nothing more. 
```